iron-request.html

```diff
@@ -243,6 +244,7 @@
       // returns. Hence, caching it as `xhr._responseType`.
       xhr.responseType = xhr._responseType = (options.handleAs || 'text');
       xhr.withCredentials = !!options.withCredentials;
+      xhr.timeout = options.timeout;
```

Comment:
Seems so. I haven't had good luck tracking down a support matrix for individual XHR features, but the test passes in all of our supported browsers.